<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="userTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="systemTextEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>211</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
